targets/targets.json

```diff
@@ -7966,6 +7966,10 @@
                 "value": 1
             }
         },
+        "components_add": [
+            "QSPIF",
```

Comment:
To remove as long as tests are not passed?

Reply:
done